src/runtime/hexagon_remote/halide_hexagon_remote.idl

```diff
@@ -31,7 +31,18 @@ interface halide_hexagon_remote
     // Retrieve the current profiling Func ID
     long poll_profiler_state(rout long func, rout long threads);
 
-    // Two more ways for powering on HVX
+    // Set a performance target.
+    long set_performance_mode(in long mode);
+    long set_performance(in long set_mips,
+                         in unsigned long mipsPerThread,
+                         in unsigned long mipsTotal,
+                         in long set_bus_bw,
+                         in unsigned long bwMegabytesPerSec,
+                         in unsigned long busbwUsagePercentage,
+                         in long set_latency,
+                         in long latency);
+
+    // Deprecated APIs
     long power_hvx_on_mode(in long mode);
```

Comment:
Yes.  We are OK with that behavior.  The power_hvx_on_mode/perf interfaces haven't been there that long, and the group that requested them will be fine with the new interfaces, and they are still just evaluating using Halide right now...so right now would be a good opportunity to remove them completely.